pkg/tiller/release_update_test.go

```diff
@@ -285,3 +285,16 @@ func TestUpdateReleaseNoChanges(t *testing.T) {
 		t.Fatalf("Failed updated: %s", err)
 	}
 }
+
+func compareStoredAndReturnedRelease(t *testing.T, rs ReleaseServer, res services.UpdateReleaseResponse) *release.Release {
+	storedRelease, err := rs.env.Releases.Get(res.Release.Name, res.Release.Version)
```

Comment:
might want to add `t.Helper()` here so we get the correct line number when the test fails. This is a recent new feature in Go 1.9.

https://golang.org/pkg/testing/#B.Helper

Reply:
Once minimum Go version requirements are moved to 1.9 , it can be added.

Reply:
#2990 does that in doc form, but this isn't a blocker for this PR.